src/stylesheets/components/_service.scss

```diff
@@ -1,3 +1,46 @@
+$hover-text-color: #fff;
+$hover-bg-color: #00897b;
+
 .service {
-  
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  padding: 50px 10px;
+  background-color: v(bg-clr);
+
+  &:hover {
+    background-color: $hover-bg-color;
+    color: $hover-text-color;
+    box-shadow: 0px 5px 20px -10px black;
+    & .service__read-more {
+      border: 2px solid $hover-text-color;
+    }
+  }
+
+  &__icon {
+    width: 60px;
+    margin-bottom: 30px;
+  }
+
+  &__heading {
+    max-width: 270px;
+    font-size: v(heading-font-size);
```

Comment:
Note (there is no any mistake): talking about other things beyond of colors, it's not as useful to use css-variables for them, but there is also no something bad.